index.Rmd

```diff
@@ -1,105 +1,228 @@
+band4 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b04.tiff")
+band5 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b05.tiff")
+band6 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b06.tiff")
+band7 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b07.tiff")
+band8 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b08.tiff")
+band9 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b09.tiff")
+```
+
+Show result of band1
+
 ```{r}
-1+2
+print(band1)
 ```
 
-Load any required packages in a code chunk (you may need to install some packages):
+Now, I check the size of image before combining them together using STACK function to make sure the sizes match. The code used "res(bandX), where X is the number of the band" shows that all bands have the same size 20x20.
+
+
+For image classification or image processing to map water or volatiles materials in selected sites, it is important to resize some image using for example the function "bandX <- aggregate(bandx, fact = 2)" in case they don't have the same size to make sure all images have the same size before adding them together using stack function in order to have a single data containg all informations from each band.
+However, for Lyot crater"V00823007" data, we observed that all bands have the same size. So we just add them all together later without resizing any band.
+
+
+We add all bands together to have a metadata containing all informations we are looking for and which will be helpful to continue the project.
+
+Most of the time, when dealing with or Earth data, researchers usually  add only band2, band3 and band7 together because those wavelength are able to reflect the distribution of water.
+
+However, we are dealing with the Martian data in our project and we observed that for Martian data, we have the same size so we will just add them all together.But I add only band4, band5, band6, band8 and band9 together because those wavelength are able to reflect the distribution of water on the selected site.
+```{r}
+#image1 <- stack(band4, band5, band6)
+image2 <- stack(band4, band5, band6, band8, band9) #Here is for Mars images
```

Comment:
There is only mars now, right?  So you can revise language to remove mention of earth.